include/circt/Dialect/OM/Evaluator/Evaluator.h

```diff
@@ -337,6 +337,27 @@ struct PathValue : EvaluatorValue {
   StringAttr field;
 };
 
+/// An unknown value, i.e., a value that we don't care about evaluating.  During
+/// evaluation, this acts like a "poison" value which will cause any operation
+/// it is folded with to become an invalid value.  The use for this is to
+/// "partially evaluate" a object where you don't care about values for all
```

Comment:
This codebase has mentions and code mentioning "partially evaluated" already, which looks to be usual need for a "thunk" in evaluators (?), but anyway just tiny note without helpful suggestion to avoid clashing with that notion of partially evaluated?

Reply:
I cleaned up the language in: b21a9f224